ukat/mapping/b0.py

```diff
@@ -94,6 +94,17 @@ def __init__(self, pixel_array, echo_list, affine, mask=None,
             self.phase_difference = self.phase1 - self.phase0
             # B0 Map calculation
             self.b0_map = self.phase_difference / (2 * np.pi * self.delta_te)
+            # Check if the full cycle exists and if so, correct the output
+            # B0 Map based on the dynamic range
+            central_pixel_value = np.mean(self.b0_map[int(self.shape[0] / 2),
```

Comment:
To avoid the issues associated with taking a single column of values from the spine as the ground truth, zero offset, perhaps take the central x% of voxels where x is tuned to include a reasonable amount of kidney (and liver and spleen)? Something like below which uses the central 50% of voxels?

```python
prop = 0.5
central_pixel_value = np.mean(self.b0_map[int(self.b0_map.shape[0] / 2 - prop / 2 * self.b0_map.shape[0]):int(self.b0_map.shape[0] / 2 + prop / 2 * self.b0_map.shape[0]), int(self.b0_map.shape[1] / 2 - prop / 2 * self.b0_map.shape[1]):int(self.b0_map.shape[1] / 2 + prop / 2 * self.b0_map.shape[1]), :])
```
Also maybe this could be renamed to `mean_central_b0` or something similar, `central_pixel_value` isn't especially descriptive and it's a mean of values rather than a single value anyway.